pureskillgg_dsdk/tome/constants.py

```diff
@@ -2,20 +2,19 @@
 import structlog
 
 
-def get_tome_manifest_key_fs(path):
-    return os.path.join(path, "tome")
-
-
-def get_tome_path_fs(root_path, prefix, tome_name):
-    tome_parent_path = (
-        root_path if prefix is None else os.path.join(root_path, *prefix.split("/"))
+def get_tome_manifest_key_fs(path, prefix, ds_type, tome_name, is_copied_header=False):
+    return os.path.join(
+        path,
+        prefix if prefix is not None else "",
+        ds_type,
+        tome_name,
+        "header" if is_copied_header else "",
+        "tome",
     )
-    return os.path.join(tome_parent_path, tome_name)
 
 
 def get_page_key_fs(path, subtype, page):
```

Comment:
We should reserve `key` for non-io stuff, so lets name this `get_page_path_fs`

Reply:
kk